Content.Server/Chemistry/ReagentEffects/PlantMetabolism/PlantPhalanximine.cs

```diff
@@ -0,0 +1,36 @@
+using Content.Server.Botany.Components;
+using Content.Server.Botany.Systems;
+using Content.Shared.Chemistry.Reagent;
+using JetBrains.Annotations;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+using Robust.Shared.Toolshed.TypeParsers;
+
+namespace Content.Server.Chemistry.ReagentEffects.PlantMetabolism
+{
+    [UsedImplicitly]
+    [DataDefinition]
+    public sealed partial class PlantPhalanximine : ReagentEffect
+    {
+        public override void Effect(ReagentEffectArgs args)
+        {
+            if (!args.EntityManager.TryGetComponent(args.SolutionEntity, out PlantHolderComponent? plantHolderComp)
+                                    || plantHolderComp.Seed == null || plantHolderComp.Dead ||
+                                    plantHolderComp.Seed.Immutable)
+                return;
+
+
+            var plantHolder = args.EntityManager.System<PlantHolderSystem>();
+            var random = IoCManager.Resolve<IRobustRandom>();
+            if (args.Source == null) return;
+            var source = args.Source;
+            //Need at least 5u, had to do it like this because plant metabolism is weird
+            if (source.TryGetReagent("Phalanximine", out var quantity) && quantity >= 4.00)
+            {
+                plantHolderComp.Seed.Viable = true;
+            }
+        }
+
+        protected override string? ReagentEffectGuidebookText(IPrototypeManager prototype, IEntitySystemManager entSys) => Loc.GetString("reagent-effect-guidebook-missing", ("chance", Probability));
```

Comment:
Why don't you write something here?

Reply:
As in a guidebook entry?

Reply:
Yes, why don't you describe the reagent effect?

Reply:
The other plant reagent reactions don't have a description so I didn't think to, I'll try to see what I can do after I'm done cleaning up the other stuff.

Reply:
may as well give them proper descriptions too

Reply:
I'm actually not sure if the guidebook is even coded to display effects that happen to hydroponics plants, which could be part of the reason why they don't have descriptions in the first place.

Reply:
Update: The guidebook appears to be coded in a way that makes it so that only the effects within `Metabolisms` are included in a reagent's effects tab, and plant metabolisms are ignored (see [`Content.Shared/Chemistry/Reagent/ReagentPrototype.cs`](https://github.com/space-wizards/space-station-14/blob/master/Content.Shared/Chemistry/Reagent/ReagentPrototype.cs) lines 170-183.) While modifying it to include plant metabolism effects would help make botany more understandable overall, I believe that would fit better in a new PR rather than expanding this one so much.

Reply:
Thanks for investigating. I agree we can leave this for later.